snowflake_driver.c

```diff
@@ -38,21 +45,54 @@ static int snowflake_handle_closer(pdo_dbh_t *dbh) /* {{{ */
         pefree(H, dbh->is_persistent);
         dbh->driver_data = NULL;
     }
-    snowflake_global_term();
+    snowflake_global_term(); // TODO: should be one time per process
```

Comment:
I believe these global init and term can move to pdo_snowflake.c such that they are called once per PDO init and term?